src/pages/registration/files_and_license_tab/FilesTableRow.tsx

```diff
@@ -79,6 +79,9 @@ export const FilesTableRow = ({ file, removeFile, baseFieldName, showFileVersion
                 {...field}
                 data-testid={dataTestId.registrationWizard.files.administrativeAgreement}
                 checked={field.value}
+                inputProps={{
+                  'aria-label': `${t('registration.files_and_license.administrative_agreement')} checkbox`,
```

Comment:
aria-label ska helst være det samme som label som man ser, siden en skjermlesere bruker å lese opp hvilket type felt det er uansett. Så slik det er nå vil det bli dobbelt opp med "checkbox"
```suggestion
                  'aria-label': t('registration.files_and_license.administrative_agreement'),
```

Reply:
kan være verdt å sjekke om det er enkelt å få til en `aria-labelledby` i stedet også, for å sikre at label og aria-label ikke blir forskjellig med tiden